doc/integrator/functionality.rst

```diff
@@ -4,20 +4,16 @@ Functionalities
 c2cgeoportal provides the concept of *functionality* that can be used to customize
 the application according to the user's permissions.
 
-A functionality may be associated to users through different ways:
+A functionality may be associated to users through the admin interface.
 
-1. Functionalities for anonymous users are defined through the
-   ``functionalities:anonymous`` variable in the ``vars.yaml`` configuration file.
-2. Functionalities for authenticated users are defined through the
-   ``functionalities:registered`` variable in the ``vars.yaml`` configuration file.
-3. Functionalities for specific roles are defined in the database through the administration interface.
-4. Functionalities for specific users are defined in the database through the administration interface.
+There are some technical user names `anonymous`, `registred` and `intranet`.
 
-Each level overrides the previous ones in the order indicated above.
-For example, if the user is authenticated and has associated functionalities in
-the ``user`` database table, then the ``functionalities:anonymous`` and
-``functionalities:registered`` configuration variables, as well as any
-functionality associated with his/her role, will be ignored.
+The functionalities resolution follows the following priorities::
```

Comment:
Add some explanation about the technical users? Similar the one in the db definition, something like
"The technical users are used to define default functionalities depending on the user's context."